src/pages/what-is-ebpf.js

```diff
@@ -72,7 +72,7 @@ class Description extends React.Component {
           <h1>eBPF Documentation</h1>
           <h2>What is eBPF?</h2>
     <p>
-      eBPF is a revolutionary technology with origins in the Linux kernel that can run sandboxed programs in an operating system kernel. It is used to safely and efficiently extend the capabilities of the kernel without requiring to change kernel source code or load kernel modules.
+      eBPF is a revolutionary technology with origins in the Linux kernel that can run sandboxed programs in a privileged context such as the operating system kernel. It is used to safely and efficiently extend the capabilities of the kernel without requiring to change kernel source code or load kernel modules.  eBPF stands for "extended Berkeley Packet Filter" but the expansion is now a misnomber since eBPF is not just for packets, and so we simply use the term eBPF today.
```

Comment:
Omitting the expansion just results in feedback asking what it stands for.  If we don't think the expansion helps, we should explicitly say it doesn't stand for anything.  Personally, I think the historical note is useful though.   For comparison, https://en.wikipedia.org/wiki/Berkeley_Packet_Filter has a discussion of the expansion, though we could update that if needed.

Reply:
A [simple search](https://www.google.com/search?q=what%20does%20ebpf%20stand%20for) will reveal this trivia if anyone is really curious about the historical background. However, explaining that it's "extended Berkeley Packet Filter" only gives a superficial (and misleading) notion of what eBPF is. If you know what BPF is, then eBPF is different enough that you need to throw away a lot of your notions about what it means; and if you don't know BPF then now you can expand the acronym into something else that adds complexity to the description but doesn't describe the goals or use for eBPF.

I'm OK with saying it doesn't stand for anything as well, but when I read through the four overall paragraphs on that first section of the page, I think that this trivia is just not that relevant to the ideas that the page conveys. Crucially, as the first paragraph on the page it's also more likely to provide first impressions. To me, this means that ideally it should express the notion of the technology in as simple terms as possible and then hook the reader in to read more to understand the technology itself rather than the historical background.

Another alternative would be to do something like include it in a glossary or FAQ lower down in the page.

Reply:
@joestringer updated to say it doesn't stand for anything.  How about now?